migration/incremental/001-extension.sql

```diff
@@ -12,7 +12,7 @@ CREATE OR REPLACE FUNCTION _prom_ext."gapfilldeltatransition_in"(
 ) RETURNS _prom_ext.GapfillDeltaTransition /* promscale::aggregates::gapfill_delta::GapfillDeltaTransition */
 IMMUTABLE PARALLEL SAFE STRICT
 LANGUAGE c /* Rust */
-AS '$libdir/promscale-0.5.0', 'gapfilldeltatransition_in_wrapper';
```

Comment:
I suspect changes in the SQL scripts are manual and that makes me pretty uncomfortable. I suggest making this a part of our `build.rs`. @JamesGuthrie WDYT?

Reply:
Yes, I'm also not quite happy with this. It's related to https://github.com/timescale/promscale_extension/issues/124, which I've been looking into this morning.